### PR TITLE
Bin omero

### PR DESCRIPTION
**Pull request by jburel**
* update default server
* use virtualenv and set ``OMERODIR``

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [TRAINING-merge#450](https://ci.openmicroscopy.org/job/TRAINING-merge/450/). See the [console output](https://ci.openmicroscopy.org/job/TRAINING-merge/450/consoleText) for more details.
Possible conflicts:
  - PR #55 pwalczysko 'idr0021 import for all workflow'
    - maintenance/scripts/link_dataset_to_new_project.py
    - maintenance/scripts/in_place_import_as.sh

--conflicts

**Comment by pwalczysko** [comment]
Hm, will be interesting how this will pan out in co-existence with the ansible workflow cc @manics @joshmoore

**Comment by joshmoore** [comment]
@pwalczysko : my assumption is that the "drop bin/omero" strategy will "just work". i.e. if you set OMERODIR and activate the virtualenv, then all is well. There's the _additional_ `/opt/omero/server/OMERO.server/bin/omero` ansible wrapper (which exists no where else) that still needs discussing but it should _also_ work.

File: maintenance/scripts/in_place_import_as.sh [hunk 1]
```diff
@@ -30,25 +30,30 @@
...
+HOST=${HOST:-workshop.openmicroscopy.org}
 FOLDER=${FOLDER:-siRNAi-HeLa}
 NUMBER=${NUMBER:-50}
 OMEUSER=${OMEUSER:-user}
 DATATYPE=${DATATYPE:-dataset}
 IMPORTTYPE=${IMPORTTYPE:-bulk}
 BULKFILE=${BULKFILE:-idr0021-scripts/idr0021-experimentA-bulk.yml}
 PROJECTNAME=${PROJECTNAME:-idr0021}
+
+export $OMERODIR
```
Comment (pwalczysko) [inline maintenance/scripts/in_place_import_as.sh]: This seems to fail

```
[importer1@ome-training-4 in-place-import]$ HOST=ome-training-4.openmicroscopy.org PASSWORD=testing NUMBER=2 IMPORTTYPE=nornal bash in_place_import_as.sh 
Starting
in_place_import_as.sh: line 45: export: `/opt/omero/server/OMERO.server': not a valid identifier
Created session for user-1@ome-training-4.openmicroscopy.org:4064. Idle timeout: 10 min. Current group: Lab1
Created session for user-2@ome-training-4.openmicroscopy.org:4064. Idle timeout: 10 min. Current group: Lab1
Finishing
[importer1@ome-training-4 in-place-import]$ 
```

Comment (jburel) [reply maintenance/scripts/in_place_import_as.sh]: ok thanks I will try that later

Comment (joshmoore) [reply maintenance/scripts/in_place_import_as.sh]:
```suggestion
export OMERODIR
```

**Comment by jburel** [comment]
@pwalczysko everything should now be sorted. Silly cut & paste

**Comment by pwalczysko** [comment]
Tested the ``in_place_import_as.sh`` script throughly on ome-training-4. All works fine.

Ready to merge fmpov

**Pull request merged**